README.md

```diff
@@ -20,14 +20,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 5. If not, file a new issue here: https://github.com/tobspr/shapez.io/issues/new
 6. I will then have a look (This can take days or weeks) and convert it to trello, and comment with the link. You can then vote there ;)
 
-## Building
-
--   Make sure `ffmpeg` is on your path
--   Install Node.js and Yarn
--   Install Java (required for textures)
--   Run `yarn` in the root folder
--   Cd into `gulp` folder
--   Run `yarn` and then `yarn gulp` - it should now open in your browser
+## Building (for windows)
+
+(credit to Hyperion-21)
+
+**SETUP**
+1. Download FFmpeg at https://www.gyan.dev/ffmpeg/builds/ffmpeg-release-full.7z
+2. Extract the .zip onto desktop, rename it to FFmpeg
+3. Move the folder into `C:\Program Files (x86)`, anywhere works but there is convienient
+4. Go to `Control Panel\System and Security\System` and click on `Advanced system settings`
+5. Go to the `Advanced` tab. Click on `Enviornment Variables...`
+6. Click on the listing that says `PATH` and click edit. 
+- On win10, you should see a list. Click `New` then type in `C:\Program Files (x86)\FFmpeg\bin`
+- On win7, you should see a window that says `Edit User Variable.` In the text field that says "Variable value:" you want to type at the end `;C:\Program Files (x86)\FFmpeg\bin`. The semicolon is for seperating the file path from the other file paths in that field.
+7. FFmpeg should now be installed. To test if you did this correctly, run `cmd.exe` as administrator and type `ffmpeg -version` and it should spit at you several lines of code.
+8. Install `Node.js`, `Yarn`, and `GitHub CLI`. All three of these softwares use setup wizards, so installation should be easy. If you don't know what setting to put in, use the default. Also make sure you have `Java` and `Git`, but both are very commom afaik.
+9. Run `cmd.exe` or `Powershell`.
+10. `cd C:\Program Files (x86)\GitHub CLI` then `gh`. Login through that, selecting HTML over SSH.
```

Comment:
You probably shouldn't clone the repo into the GitHub CLI files (and also the below instructions use something else).